privaterelay/templates/includes/dashboard-filter.html

```diff
@@ -22,80 +22,82 @@
     <div class="c-filter-date hidden">
         <div class="c-filter-icon c-filter-icon-calendar"></div>
     </div>
-    <div class="c-filter-category">
+    <div class="c-filter-category {% if not user_profile.has_unlimited %} hidden {% endif %}">
```

Comment:
Should this be `is-hidden`?

Reply:
Yes, but I'm leaving as is for the moment. Hot-fix'in!